suitcase/hdf5.py

```diff
@@ -94,8 +95,20 @@ def export(headers, filename, stream_name=None, fields=None, timestamps=True, us
                                                 compression='gzip',
                                                 fletcher32=True)
                     data = [e['data'][key] for e in events]
-                    dataset = data_group.create_dataset(
-                        key, data=data, compression='gzip', fletcher32=True)
+                    try:
+                        # save numerical data
+                        dataset = data_group.create_dataset(
+                            key, data=data, compression='gzip', fletcher32=True)
+                    except TypeError:
+                        try:
+                            # save data with str type, or list of str
+                            data = np.array(data).astype('|S9')
```

Comment:
This will truncate at 9 letters.  Should probably cast up fixed length at what ever the longest string is.